<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Статистика</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_stats">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести статистику</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="period">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>Период:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>771</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QTableWidget" name="income_table"/>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="visits_table"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Доход:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTableWidget" name="typies_table"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Статистика по посещениям:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Статистика по митапам:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>771</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Постоянные покупатели:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="reg_table"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
